tests/processes/LogPrinterThreadTest.py

```diff
@@ -2,14 +2,10 @@
 import unittest
 
 from coala_utils.ContextManagers import retrieve_stdout
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.processes.LogPrinterThread import LogPrinterThread
 
 
-class TestPrinter(LogPrinter):
-
-    def __init__(self):
-        LogPrinter.__init__(self, self)
+class TestPrinter():
```

Comment:
It is actually covering branch https://github.com/coala/coala/pull/4320/files/c367b4ca2e3157746a51ed3ad13abdbfeabb21ec#diff-eea0f007c4790db0687a6e6f592f8f56R23